src/1-foundations/2-homotopy-type-theory/01-types-are-higher-groupoids.rzk.md

```diff
@@ -5,3 +5,141 @@ This is a literate Rzk file:
 ```rzk
 #lang rzk-1
 ```
+
+Groupoids are categories in which all morphisms are isomorphisms.
+Alternativerly, groupoids can be viewed as a generalization of groups, where not all pairs of elements can be composed
+(but the group laws for the operation hold).
+
+## Path symmetry
+!!! lemma "Lemma 2.1.1."
```

Comment:
`lemma` is not a supported admonition type. See [MkDocs Material Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) for reference.